src/grasshopper/lib/fixtures/grasshopper_constants.py

```diff
@@ -23,6 +23,8 @@ class GrasshopperConstants:
         "slack_report_failures_only",
         "cleanup_s3",
         "rp_token",
+        "rp_uuid",
+        "rp_launch",
```

Comment:
why so we have `rp_launch` and r`p_launch_name`? is one legacy?

Reply:
yes, one is legacy